src/argilla/client/feedback/dataset/base.py

```diff
@@ -66,63 +65,8 @@ class FeedbackDataset(ArgillaDatasetMixin, HuggingFaceDatasetMixin):
-
-    Examples:
-        >>> import argilla as rg
-        >>> rg.init(api_url="...", api_key="...")
-        >>> dataset = rg.FeedbackDataset(
-        ...     fields=[
-        ...         rg.TextField(name="text", required=True),
-        ...         rg.TextField(name="label", required=True),
-        ...     ],
-        ...     questions=[
-        ...         rg.TextQuestion(
-        ...             name="question-1",
-        ...             description="This is the first question",
-        ...             required=True,
-        ...         ),
-        ...         rg.RatingQuestion(
-        ...             name="question-2",
-        ...             description="This is the second question",
-        ...             required=True,
-        ...             values=[1, 2, 3, 4, 5],
-        ...         ),
-        ...         rg.LabelQuestion(
-        ...             name="question-3",
-        ...             description="This is the third question",
-        ...             required=True,
-        ...             labels=["positive", "negative"],
-        ...         ),
-        ...         rg.MultiLabelQuestion(
-        ...             name="question-4",
-        ...             description="This is the fourth question",
-        ...             required=True,
-        ...             labels=["category-1", "category-2", "category-3"],
-        ...         ),
-        ...     ],
-        ...     guidelines="These are the annotation guidelines.",
-        ... )
-        >>> dataset.add_records(
-        ...     [
-        ...         rg.FeedbackRecord(
-        ...             fields={"text": "This is the first record", "label": "positive"},
-        ...             responses=[{"values": {"question-1": {"value": "This is the first answer"}, "question-2": {"value": 5}}}],
-        ...             external_id="entry-1",
-        ...         ),
-        ...     ]
-        ... )
-        >>> dataset.records
-        [FeedbackRecord(fields={"text": "This is the first record", "label": "positive"}, responses=[ResponseSchema(user_id=None, values={"question-1": ValueSchema(value="This is the first answer"), "question-2": ValueSchema(value=5), "question-3": ValueSchema(value="positive"), "question-4": ValueSchema(value=["category-1"])})], external_id="entry-1")]
-        >>> dataset.push_to_argilla(name="my-dataset", workspace="my-workspace")
-        >>> dataset.argilla_id
-        "..."
-        >>> dataset = rg.FeedbackDataset.from_argilla(argilla_id="...")
-        >>> dataset.records
-        [FeedbackRecord(fields={"text": "This is the first record", "label": "positive"}, responses=[ResponseSchema(user_id=None, values={"question-1": ValueSchema(value="This is the first answer"), "question-2": ValueSchema(value=5), "question-3": ValueSchema(value="positive"), "question-4": ValueSchema(value=["category-1"])})], external_id="entry-1")]
     """
 
-    argilla_id: Optional[UUID] = None
-
```

Comment:
If a user is using the property, this change may break the code. Maybe just a property with 2 different implementations?

```python
@property
def argilla_id() -> UUID
   raise NotImplementedError()

# for remote dataset:
def argilla_id() -> UUID:
   warnings.warning("use self.id instead...")
   return self.id

# for local dataset
def argilla_id() -> UUID:
   warnings.warning("local datases don't track remote argilla info...")
   return None
```